draft-ietf-tls-tls13.md

```diff
@@ -2335,7 +2335,7 @@ extensions
 : A list of extensions.
 {:br }
 
-###  Certificate Request
+###  Certificate Request {#certificate-request}
```

Comment:
You don't need to add this anchor here; every section gets an autogenerated anchor named in this exact form, so this is a no-op. Manually specifying these is only useful if you've got a long section name or otherwise wish to use an abbreviation, instead.